content/docs/get_started/login_requiring.md

```diff
@@ -0,0 +1,163 @@
+---
+$title: Create a login-requiring AMP page
+$order: 3
+---
+Some user interactions with a page, such as leaving a comment, could be conditioned by a login flow. A login flow can be implemented with AMP using the [amp-access](https://www.ampproject.org/docs/reference/components/amp-access) component combined with the [amp-form](https://www.ampproject.org/docs/reference/components/amp-form) component. 
+To see a sample implementation, visit the [comment section sample](https://ampbyexample.com/samples_templates/comment_section/) at [ampbyexample.com](https://ampbyexample.com).
+
+The [comment section sample](https://ampbyexample.com/samples_templates/comment_section/) combines `amp-access` and `amp-form` to realize a comment section which is enabled only when an user has logged in. In order to explain how this sample works, we will follow the set of actions that will be performed once you land on the page.
+
+## Login
+
+The first time you land on the page, you can see 2 comments and a login button. 
+
+<amp-img src="/static/img/login-button.png" alt="Login button" height="290" width="300"></amp-img>
+
+If you look for the login button in the code, you will find:
+
+[sourcecode:html]
+<span amp-access="NOT loggedIn" role="button" tabindex="0" amp-access-hide>
+          <h5>Please login to comment</h5>
+          <a on="tap:amp-access.login-sign-in" class="button button-primary comment-button">Login</a>  
+</span> 
+[/sourcecode]
+
+The span element is conditioned by the `amp-access` attribute with the value `NOT logged in`. 
+
+Here it follows the configuration for amp-access:
+
+[sourcecode:html]
+<script id="amp-access" type="application/json">
+  {
+    "authorization": "https://ampbyexample.com/samples_templates/comment_section/authorization?rid=READER_ID&url=CANONICAL_URL&ref=DOCUMENT_REFERRER&_=RANDOM",
+    "noPingback": "true",
+    "login": {
+      "sign-in": "https://ampbyexample.com/samples_templates/comment_section/login?rid=READER_ID&url=CANONICAL_URL",
+      "sign-out": "https://ampbyexample.com/samples_templates/comment_section/logout"
+    },
+    "authorizationFallbackResponse": {
+      "error": true,
+      "loggedIn": false
+    }
+  }
+</script>
+[/sourcecode]
+
+The authorization endpoint is deployed as part of AMPByExample. It’s in fact responsibility of the publisher of the page to provide this endpoint. In the sample case, for simplicity sake, we implemented a simple logic where when this request is received, the server reads the value of a cookie called `ABE_LOGGED_IN`. If the cookie is not there, we return a json containing `loggedIn = false`. This means that the first time you land on the page, this request will return `loggedIn = false` and the login button will be shown.
```

Comment:
"we return a json" => "we return a JSON response".

"This means that the first time you land on the page" => "As result, the first time a user lands on the page"